Assignments/SCF-Session-2/hexadecimalCalc/HexCalc.java

```diff
@@ -0,0 +1,277 @@
+package hexadecimalCalc;
+
+import java.util.Scanner;
+
+class Functions
+{
+	public String add(String hexNum1, String hexNum2)
+	{
+		int decNum1 = hexToDec(hexNum1);
+		int decNum2 = hexToDec(hexNum2);
+		
+		int decAdd = decNum1 + decNum2;
+		String hexAdd = decToHex(decAdd);
+		
+		return hexAdd;
+		
+	}
+	
+	public String substract(String hexNum1, String hexNum2)
+	{
+		int decSub = 0;
+		int decNum1 = hexToDec(hexNum1);
```

Comment:
If the variables have the same data type then no need to put the data type with each variable. Above can done like 
`int decSub = 0,
decNum1 = hexToDec(hexNum1),
decNum2 = hexToDec(hexNum2);`

Reply:
done